recipes/srm-ifce/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "srm-ifce" %}
+{% set version = "1.24.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.cern.ch/dmc/{{ name }}/-/archive/v{{ version }}/{{ name }}-v{{ version }}.tar.gz
+  sha256: 9bfea03b33c387a37ff454aaed936ac8f17007b92ed6693c790a8365c2c59462
+
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - pkg-config
+    - sed
+  host:
+    - glib
+    - gsoap =2.7
+    - openssl 1.0.2r
```

Comment:
I'm using openssl 1.0 as I'm building something which needs it so I'll migrate over to 1.1 in the feedstock.

I've checked and run_export is used correctly in openssl=1.0.2r.